wlib/RepositoryTreeNode.cpp

```diff
@@ -317,6 +317,17 @@ int CAttributeNode::GetDispImage()
 	case SYNTAX_CHECK_FAIL:
 		return IID_DOCUMENT_SYNTAXFAIL;
 	}
+
+	std::wstring ext = m_attribute->GetType()->GetFileExtension();
```

Comment:
Should use "GetRepositoryCode" rather than "GetFileExtension".  You can then also move the ```#define ATTRIBUTE_TYPE_*``` to the header for comparison.